### PR TITLE
OSSRDBMS Swagger completeness related changes

### PR DESCRIPTION
**Pull request by shinilm**
### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [X ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [X ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [X ] SDK of Go
      - [ ] PowerShell
      - [X ] CLI
      - [X ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [X ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by JeffreyRichter — APPROVED** [review]

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/examples/RecoverableServersGet.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "name": "recoverableServers",
+        "type": "Microsoft.DBforMariaDB/servers/recoverableServers",
+        "properties": {
+          "edition": "GeneralPurpose",
+          "serviceLevelObjective": "GP_Gen5_2",
+          "vCore": 2,
+          "hardwareGeneration": "Gen5",
+          "lastAvailableBackupDateTime": "2020-11-20T01:06:29.78Z",
+          "version": "10.3"
+        }
```
Comment (j5lim) [inline specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/examples/RecoverableServersGet.json:21]: List API must return an array. Is this really what the API returns?

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/mariadb.json [hunk 2]
```diff
@@ -1072,6 +1130,93 @@
...
+        "tags": [
+          "RecoverableServers"
+        ],
+        "operationId": "RecoverableServers_Get",
+        "x-ms-examples": {
+          "ReplicasListByServer": {
+            "$ref": "./examples/RecoverableServersGet.json"
+          }
+        },
+        "description": "Gets a recoverable MariaDB Server.",
```
Comment (j5lim) [inline specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/mariadb.json:1144]: It should be "List" even if it returns a singleton resource.

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2018-06-01/mariadb.json [hunk 3]
```diff
@@ -1186,6 +1244,93 @@
...
+        "tags": [
+          "RecoverableServers"
+        ],
+        "operationId": "RecoverableServers_Get",
+        "x-ms-examples": {
+          "ReplicasListByServer": {
+            "$ref": "./examples/RecoverableServersGet.json"
+          }
+        },
+        "description": "Gets a recoverable MariaDB Server.",
```
Comment (j5lim) [inline specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2018-06-01/mariadb.json:1258]: Same comments as the preview version.

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/examples/RecoverableServersGet.json (hunk 1 above)
Comment (shinilm) [reply specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/examples/RecoverableServersGet.json:21]: Yes this is what the service returns

This changes are to align with the service for S360 violations.  The service has been in GA for 3 years and there are no changes at the service side and we are just adding the missing API back to the swagger.  These API were only required from the portal perspective and no customers consuming this.  We are not doing any future development in this platform and is in the sunset path.

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/mariadb.json (hunk 2 above)
Comment (shinilm) [reply specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/mariadb.json:1144]: This is what actually service returns

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2018-06-01/mariadb.json (hunk 3 above)
Comment (shinilm) [reply specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2018-06-01/mariadb.json:1258]: This is what service returns

**Review by j5lim — APPROVED** [review]
This is to make old api version and swagger aligned. Signing from ARM side.

**Comment by shinilm** [comment]
Hi Erich,

The ARM team and the Breaking change board approved the PR.  Please help merge  this PR.

Thanks
Shinil

**Pull request merged**